### DISCUSSION TITLE
Grade Vue for other counties?

### DISCUSSION THREAD

**Discussion by JaytirthJOSHI** [PurelyAnecdotal/gradecompass#28] [Q&A] [answered]
How should I use it for Cobb parentvue.cobbk12.org

**Comment by PurelyAnecdotal** [comment, marked as answer]
You can use Gradevue with other school districts by clicking the "Advanced" dropdown on the Sign In screen and setting "Domain" to the domain of your Synergy website (in your case, `parentvue.cobbk12.org`).

If you need to create a password, you'll also want to go to the password reset page of [your school district](https://parentvue.cobbk12.org/PXP2_Password_Help.aspx) instead.
